cypress/integration/basic-functionalities/local-tests/reporter/1_Create_a_report.spec.js

```diff
@@ -0,0 +1,90 @@
+      cy.get('[data-test-id="button-create-report"]').click(); // We click on the New report button
+      cy.get('[data-test-id="button-save"].disabled'); // Since there is no data enetered, the save button is disabled and if disabled the test should pass
+    });
+  });
+
+  context("Test for creating a report with one missing required fild", () => {
+    it("shouldn't be able to create a report with missing name", () => {
+      cy.get('[data-test-id="input-handle"]').type("handle");
+      cy.get('[data-test-id="button-save"].disabled'); // Since there is no name enetered, the save button is disabled and if disabled the test should pass
+    });
+    it("shouldn't be able to create a report with missing handle", () => {
+      cy.get('[data-test-id="input-handle"]').clear();
+      cy.get('[data-test-id="input-name"]').type("Name");
+      cy.get('[data-test-id="button-save"].disabled'); // Since there is no handle enetered, the save button is disabled and if disabled the test should pass
+    });
+  });
+
+  context("Test for creating a report with misconfigured handle", () => {
+    it("shouldn't be able to create a report with incorrect handle", () => {
+      cy.get('[data-test-id="input-handle"]').type("h"); // We just type one letter and the validation should fail
+      cy.get('[data-test-id="button-save"].disabled'); // Since there is incorrect handle enetered, the save button is disabled and if disabled the test should pass
+    });
+  });
+
+  context("Test for checking if the new report, permissions and delete buttons aren't displayed", () => {
+    it("shouldn't be displayed when into the new report view", () => {
+      cy.get('[data-test-id="button-create-report"]').should("not.exist"); // Here we check that the New report button is not present
+      cy.get('[data-test-id="button-permissions"]').should("not.exist"); // Here we check that the Permissions button is not present
+      cy.get('[data-test-id="button-delete"]').should("not.exist"); // Here we check that the Delete button is not present
+    });
+  });
+
+  context("Test for creating standard report using the create report button", () => {
+    it("should be able to create a report", () => {
+      cy.get('[data-test-id="button-back"]').click(); // We click back in order to go on the main page of reporter
+      cy.get('[data-test-id="button-create-report"]').click(); // We click on New report button
+      cy.get('[data-test-id="input-name"]').type("Cypress report"); // We enter the name of the report
+      cy.get('[data-test-id="input-handle"]').type("cypress_report"); // We enter the handle of the report
+      cy.get('[data-test-id="input-description"]').type("This is an automated description."); // We enter the description
+      cy.get('[data-test-id="button-save"]').click(); // We click on the Save button
+      cy.get('[data-test-id="button-back"]').click(); // We click on the Back button to go on the main page
+     });
```

Comment:
After successfully creating a report, please, do two more things:
- see if the b-toast-success message is shown
- go to the list of reports, open the same report you've just created and make sure an error message doesn't pop up. If it does then there's something wrong with the report's configuration